Update onpremisespublishing.md (#18942)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/applicationsegment.md

---
title: "applicationSegment resource type"

> description: "Represents applicationsegment object when publishing an on-premises wildcard application with Azure AD Application Proxy."

Reviewer: ```suggestion
description: "An abstract type that exposes properties used to configure segment configurations that are allowed for an on-premises wildcard application that's published through Azure AD Application Proxy."
```

Author: Updated as suggested

ms.localizationpriority: medium
author: "dhruvinshah"
ms.prod: "applications"
doc_type: "resourcePageType"
---

# applicationSegment resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> This is an abstract type noting the application segment types when publishing an on-premises wildcard application with Azure AD Application Proxy. The following types are derived from this abstract type:

Reviewer: For the first sentence, apply what you apply in Line 3

Author: done

+ [webaApplicationSegment](webapplicationsegment.md)

## Properties

| Property     | Type        | Description |
|:-------------|:------------|:------------|

> |id|String|The unique identifier that is assigned to aa application segment by Azure AD. Not nullable. Read-only. Supports `$filter` (`eq`).  |

Reviewer: ```suggestion
|id|String|The unique identifier that is assigned to an application segment by Azure AD. Not nullable. Read-only. Supports `$filter` (`eq`).  |
```

## JSON representation

The following is a JSON representation of the resource.
<!-- {
  "blockType": "resource",
  "@odata.type": "microsoft.graph.applicationSegment"
}
-->
``` json
{
  "@odata.type": "microsoft.graph.applicationSegment",

>   "id": "String"

Reviewer: ```suggestion
  "id": "String (identifier)"
```
